pkgs/os-specific/linux/apfs/default.nix

```diff
@@ -5,20 +5,18 @@
   kernel,
   kernelModuleMakeFlags,
   nixosTests,
+  gitUpdater,
 }:
 
-let
-  tag = "0.3.14";
-in
-stdenv.mkDerivation {
+stdenv.mkDerivation (finalAttrs: {
   pname = "apfs";
-  version = "${tag}-${kernel.version}";
+  version = "0.3.15";
```

Comment:
Please don't do this on a case-by-case basis and with the motivation being to appease other tools failing to handle this.